recipes-extended/xen/files/0001-convert-block-tap-for-shared.patch

```diff
@@ -428,6 +428,8 @@ Subject: [PATCH] convert block-tap for shared
 +#                requested mode; or
 +#    'ok':       the device may be used.
 +#
++#    '$n':       Number of instances using the tapdev
```

Comment:
I just realized that commit ```c8547eed43cd1764b87701b1b47dfe8a6fb327c5``` " xen: convert-block-tap-for-shared.patch " makes changes to the block scripts, I think it could probably be squashed into the original commit?

Reply:
I think all convert-block-tap-for-shared.patch changes should be squashed together since they are fixes and we agreed on the xenmgr env var approach for TAPDISK3_CRYPTO_KEYDIR.